test/features/ex/errors.js

```diff
@@ -1,89 +0,0 @@
-'use strict';
```

Comment:
What's wrong with these tests?

Reply:
I can't tell what these tests have to do with kartotherian.

Reply:
These are probably carried over from `service-template-node`, so up to you whether to leave these or not, they're checking the more generic service functionality. Up to you whether to leave them or not, more tests is always better

Reply:
> more tests is always better
I agree in general but testing 3rd parties for behavior you don't depend on increases the maintenance burden for no reason. 

Like I said before, maybe these are important somehow but I really don't understand what they are about and how they can possibly relate to kartotherian.